convert2rhel/unit_tests/utils_test.py

```diff
@@ -347,3 +348,79 @@ def test_prompt_user(question, is_password, response, monkeypatch):
         monkeypatch.setattr(moves, "input", lambda _: response)
 
     assert prompt_user(question, is_password) == response
+
+
+@pytest.mark.parametrize(
+    ("content", "output"),
+    (
+        (
+            "[subscription_manager]\npassword = correct_password",
+            {"password": "correct_password", "activation_key": None},
+        ),
+        (
+            "[subscription_manager]\nactivation_key = correct_key\nPassword = correct_password",
+            {"password": "correct_password", "activation_key": "correct_key"},
+        ),
+        (
+            "[subscription_manager]\nincorrect_option = incorrect_content",
+            {"password": None, "activation_key": None},
+        ),
+        ("[INVALID_HEADER]\nactivation_key = correct_key", {"password": None, "activation_key": None}),
+        ("nonexisting", {"password": None, "activation_key": None}),
+    ),
+)
+def test_options_from_config_files_default(monkeypatch, content, output, tmp_path, caplog):
+    """Test config files in default path."""
+    path = os.path.join(str(tmp_path), "convert2rhel.ini")
+    tool_opts = ToolOpts()
+    if content != "nonexisting":
```

Comment:
This is fine for a test but note that `None` is the usual sentinel value to use.  (And if None has meaning, then `sentinel = object()` can be used)